doogie_control/launch/robot_control.launch

```diff
@@ -4,7 +4,7 @@
     <rosparam file="$(find doogie_control)/config/wheel_control.yaml" command="load"/>
 
     <node pkg="controller_manager" type="spawner" name="spawns_control" 
-        output="screen" respawn="false" args="--namespace=/$(arg ns) --timeout 5
+        output="screen" respawn="false" args="--namespace=/$(arg ns) --timeout 20
```

Comment:
Can we get a lower timeout? Mabe 5s or even 10s.

Reply:
I don't know. We need to do some tests with the real robot because in the raspberry the startup of the nodes is slow